components/general/IconItem.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "@/styles/OurClients.module.css";
+
+interface Props {
+  aria_label: string;
+  alt: string;
+  icon: string;
+}
+
+const IconItem = ({ aria_label, alt, icon }: Props) => {
+  return (
+    <div>
```

Comment:
Do we really need `div` here？ And what is the benefit of extracting a component here?